routes/users.go

```diff
@@ -282,7 +278,7 @@ func PutUpdateUserHandler(w http.ResponseWriter, r *http.Request) {
 
 	_, err = Db.Collection("users").UpdateOne(r.Context(), queries.WithID(userID), update)
 	if err != nil {
-		log.Printf("ERROR %v", http.StatusInternalServerError)
+		log.Printf("ERROR: %v", err)
```

Comment:
but you still need to return and set the response to 500

Reply:
oh i see